### PR TITLE
Collection of Quarkus properties doesn't work with @ConfigRoot with Java source file

### LINKED ISSUE #138: Collection of Quarkus properties doesn't work with @ConfigRoot with Java source file
According @jeffmaury feedback, the collection of Quarkus properties with @ConfigRoot with Java source file (and not JARs) doesn't work. The problem is [here](https://github.com/redhat-developer/quarkus-ls/blob/master/quarkus.jdt/com.redhat.quarkus.jdt.core/src/main/java/com/redhat/quarkus/jdt/core/JDTQuarkusManager.java#L451)

### PR DESCRIPTION
**Pull request by angelozerr**
Collection of Quarkus properties doesn't work with @ConfigRoot with Java source file

Fixes #138

Signed-off-by: azerr <azerr@redhat.com>

### CODE REVIEW

**Comment by angelozerr** [comment]
To test this PR, create a Java class in a Quarkus application like this:

```
package org.acme.config;

import io.quarkus.runtime.annotations.ConfigRoot;

@ConfigRoot
public class CustomExtensionConfig {

	private String property1;

	private Integer property2;
}

```

Open completion in application.properties file, you should see the 2 properties:

 * quarkus.custom-extension.property1
 * quarkus.custom-extension.property2

File: quarkus.jdt/com.redhat.quarkus.jdt.test/src/main/java/com/redhat/quarkus/jdt/core/JDTQuarkusManagerConfigPropertyTest.java [hunk 1]
```diff
@@ -42,7 +42,13 @@ public void configQuickstartFromClasspath() throws Exception {
 		Assert.assertNotNull("Test existing of quarkus-core-deployment*.jar", f);
 
 		String expectedDeploymentJar = f.getAbsolutePath();
-		assertProperties(infoFromClasspath, 185 /* properties from JAR */ + 3 /* properties from Java sources */,
+		assertProperties(infoFromClasspath, 185 /* properties from JAR */ + 3 /*
```
Comment (fbricon) [inline quarkus.jdt/com.redhat.quarkus.jdt.test/src/main/java/com/redhat/quarkus/jdt/core/JDTQuarkusManagerConfigPropertyTest.java]: remove excess of empty lines

File: quarkus.jdt/com.redhat.quarkus.jdt.test/src/main/java/com/redhat/quarkus/jdt/core/JDTQuarkusManagerConfigPropertyTest.java [hunk 2]
```diff
@@ -80,7 +96,13 @@ public void configQuickstartFromJavaSources() throws Exception {
 		QuarkusProjectInfo infoFromJavaSources = getQuarkusProjectInfoFromMavenProject("config-quickstart",
 				QuarkusPropertiesScope.sources);
 
-		assertProperties(infoFromJavaSources, 3 /* properties from Java sources */,
+		assertProperties(infoFromJavaSources, 3 /* properties from Java sources with ConfigProperty */ + 2 /*
+																											 * properties
```
Comment (fbricon) [inline quarkus.jdt/com.redhat.quarkus.jdt.test/src/main/java/com/redhat/quarkus/jdt/core/JDTQuarkusManagerConfigPropertyTest.java]: empty lines

File: quarkus.jdt/com.redhat.quarkus.jdt.test/src/main/java/com/redhat/quarkus/jdt/core/JDTQuarkusManagerConfigPropertyTest.java (hunk 1 above)
Comment (angelozerr) [reply quarkus.jdt/com.redhat.quarkus.jdt.test/src/main/java/com/redhat/quarkus/jdt/core/JDTQuarkusManagerConfigPropertyTest.java]: @fbricon updated

File: quarkus.jdt/com.redhat.quarkus.jdt.test/src/main/java/com/redhat/quarkus/jdt/core/JDTQuarkusManagerConfigPropertyTest.java (hunk 2 above)
Comment (angelozerr) [reply quarkus.jdt/com.redhat.quarkus.jdt.test/src/main/java/com/redhat/quarkus/jdt/core/JDTQuarkusManagerConfigPropertyTest.java]: @fbricon updated

**Pull request merged**